src/index.css

```diff
@@ -36,22 +35,17 @@ header[role="banner"] {
   display: none !important;
 }
 
-/* 4. Center main content. */
-/* main > :first-child {
-  width: 100% !important;
-} */
```

Comment:
📝 Remove old, unused styles to center main content. (Content is centered by default now.)